sdk/eventhub/Azure.Messaging.EventHubs/tests/Azure.Messaging.EventHubs.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>Azure.Messaging.EventHubs.Tests</AssemblyName>
-    <VersionPrefix>5.0.0</VersionPrefix>
+    <VersionPrefix>5.0.1</VersionPrefix>
```

Comment:
Is there a reason to actually change the version of tests and samples? I'd suggest just keeping them frozen as we don't ship them anywhere.

Reply:
You know, it's a fair point.   I was arguing with myself whether to bind everything to a common version in the props file, leave them static, or just adjust and was bumping them just to kick the can down the road and avoid having to make a decision.

You're the the-breaker;  setting the non-products back to `1.0.0` and leaving them there.